templates/_configs.njk

```diff
@@ -6,8 +7,9 @@
     <section>
         <div class="mb-config-options">
             <div class="superfences-tabs">
-            <input name="__tabs_1" type="checkbox" id="__tab_1_0">
-                <label class="tab-selector" for="__tab_1_0"><i class="icon fa fa-code"></i></label>
+            {% set tabid = tabid + 1 %}
+            <input name="{{tabid}}" type="checkbox" id="{{tabid}}">
```

Comment:
Keep the prefix `__tab_`. Because there can be possibilities you will get same numeric id some where else also

Reply:
Fixed with 92fed5f2154010c5cc2350ae49d8b6d3efa4a369